src/card.css

```diff
@@ -0,0 +1,89 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
```

Comment:
I don't see paddings on the demo link, did you run deploy command one more time after you changes? If no, please run 'npm run deploy' and check that all changes will be applied on the demo link

Reply:
yheah, i did it, but ill do it one more time